Project/Source/Modules/ModuleWindow.cpp

```diff
@@ -128,6 +138,46 @@ void ModuleWindow::SetTitle(const char* title) {
 	SDL_SetWindowTitle(window, title);
 }
 
+void ModuleWindow::SetCursor(bool isPlaying) {
+	if (isPlaying) {
+		// Load resource
+		Scene* scene = App->scene->scene;
+		ResourceTexture* cursorResourceTexture = App->resources->GetResource<ResourceTexture>(scene->GetCursor());
+		if (cursorResourceTexture == nullptr) {
+			return;
+		}
+		// From glTexture to SDL_Surface
+		int w = 0;
+		int h = 0;
+		glGetTextureLevelParameteriv(cursorResourceTexture->glTexture, 0, GL_TEXTURE_WIDTH, &w);
+		glGetTextureLevelParameteriv(cursorResourceTexture->glTexture, 0, GL_TEXTURE_HEIGHT, &h);
+
+		void* pixel_data = malloc(w*h*4);
+		memset(pixel_data, 0, w*h * 4);
+
+		glBindTexture(GL_TEXTURE_2D, cursorResourceTexture->glTexture);
+		glGetTexImage(GL_TEXTURE_2D, 0, GL_RGBA, GL_UNSIGNED_BYTE, pixel_data);
+
+		SDL_Surface* surface = SDL_CreateRGBSurfaceFrom(pixel_data, w, h, 32, 4 * w, 0x000000ff, 0x0000ff00, 0x00ff0000, 0xff000000);
+
+		// Scale image
+		SDL_Surface* scaledImage = SDL_CreateRGBSurfaceWithFormat(0, scene->widthCursor, scene->heightCursor, 32, SDL_PIXELFORMAT_RGBA32);
+		SDL_SoftStretch(surface, &surface->clip_rect, scaledImage, &scaledImage->clip_rect);
```

Comment:
Sorry I didnt see that before. This breaks when cursor width and height are zero.
I would set a minimum size of 1 on the ImGui sliders in panelConfiguration.

Reply:
This theoretically can not happen, i initialize these variables to 30 and then in the ImGui I make me min is 10 max 100